### ISSUE TITLE
No more progress!

### ISSUE THREAD

**Issue by veilin** [jeremyaboyd/simplemvcjs#3]
Hi Jeremy,

I was seriously looking forward to your continuation on this project and probably every day refreshed to see what you were doing. 

I though that, as soon as you get to the saas I would like to start helping out. 

Let us know what the future holds, and looking forward to you new contributions,

Regards,
Rudolf

**Comment by jeremyaboyd** [comment]
Hey Rudolf,

It isn't abandoned, I promise! 

I took a couple weeks off to work on another project that I'm building with SimpleMVC. Some of that project will get merged back into SimpleMVC (specifically around the SimpleMVC.Membership service), but I had to use the framework in the real world to see the parts that needed more work.

There is still work happening, it is just disconnected from the repository.

I'll have something to show for it soon!

Thanks,
Jeremy
